delocate/tools.py

```diff
@@ -377,6 +377,10 @@ def dir2zip(in_dir, zip_fname):
             # Preserve file permissions, but allow copy
             info = zipfile.ZipInfo(in_fname)
             info.filename = relpath(in_fname, in_dir)
+            if os.name == 'nt':
```

Comment:
Would `os.path.sep == '\'` give the same answer?  Maybe it's more obvious?

Reply:
You mean `relpath(in_fname, in_dir).replace(os.path.sep, '/')` for all platforms?

Reply:
No, I meant:

```
if os.path.sep == '\':
```

instead of:

```
if os.name == 'nt':
```